lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.h

```diff
@@ -51,7 +51,7 @@ class MemoryReaderLocalBufferHolder {
 
 class LLDBMemoryReader : public swift::remote::MemoryReader {
 public:
-
+  static constexpr uint8_t LLDBAddressSpace = 1;
```

Comment:
Maybe add a comment here saying that anything but address space 0 is fair play and that each tool/client can define their own, so there's no risk trampling over someone else's address space.

Reply:
I'll add a comment on the stable branch